domain/schema/model/sql/0024-relation.sql

```diff
@@ -157,6 +157,8 @@ CREATE TABLE relation_sequence (
     sequence INT NOT NULL DEFAULT 0
 );
 
+INSERT INTO relation_sequence (sequence) VALUES (0);
```

Comment:
note: `relation_sequence` will be replaced with the `sequence` table in a near future PR.